client/Packages/com.beamable/Editor/BeamCli/BeamWebCommand.cs

```diff
@@ -268,6 +274,11 @@ public async Promise Run()
 				//  that is _fine_, but we need to handle it.
 				CliLogger.Log("cli server died, " + ioException.Message);
 			}
+			catch (OperationCanceledException cancelledException)
+			{
+				// A cancellation was requested so the connection was terminated
+				CliLogger.Log("cli command was cancelled, " + cancelledException.Message);

```

Comment:
does any extra explicit clean up need to happen with the http object?
```
using HttpResponseMessage response
```

Reply:
Nice call! Done!